splunk_logger.go

```diff
@@ -517,6 +524,73 @@ func (l *splunkLogger) queueMessageAsync(message *splunkMessage) error {
 	return nil
 }
 
+func telemetry(info logger.Info, l *splunkLogger, sourceType string, splunkFormat string) {
+
+	//Send weekly
+	timer := time.NewTicker(600000000000000)
```

Comment:
use constant for hours instead of large number 600000000000000
will be something like 7 * 24 * Time.day

Reply:
done